cvc5/src/cvc5_term.cpp

```diff
@@ -225,7 +229,7 @@ Op Cvc5Term::get_op() const
   if (kind2primop.find(cvc5_kind) == kind2primop.end())
   {
     throw NotImplementedException("get_op not implemented for cvc5 Kind "
-                                  + cvc5::kindToString(cvc5_kind));
+                                  + std::to_string(cvc5_kind));
```

Comment:
Does this still print the name of the kind?  Or just an integer?

Reply:
This is a good point -- maybe a test can be added to verify that the name of the kind is printed?

Reply:
I'm not sure. I can check, but `cvc5::kindToString` has been marked deprecated and will be removed in the next minor cvc5 release according to the cvc5 release notes, so I don't think we should continue using it.

Reply:
Based on the implementation here, it will print the name of the kind: `https://github.com/cvc5/cvc5/blob/58cda4cdc96a9cdc0e267ba4f4ddd00bcd653a1f/src/api/cpp/cvc5.cpp#L9018`. I think this is identical to the former `kindToString` function, they just changed the name to be more standard-conformant.

Reply:
Just verified this: it still prints the name of the sort kind.